### ISSUE TITLE
Manual Installing in Swift

### ISSUE THREAD

**Issue by jktestaccount** [Azure/azure-mobile-apps-ios-client#158]
It would be helpful if you could update your docs to let people know when importing in swift:
• A bridging header needs to be created
• The WebKit framework has to be added
• The SafariServices framework has to be added

**Comment by markosaric** [comment]
I agree with @jktestaccount the documentation is out dated. I followed instruction found on Azure and I've lost few hours to find why it didn't work and where have I made a mistake. Finally I ran to this post and it worked with Swift 4.0 and iOS 11.2.

**Comment by jonasrotilli** [comment]
Would it be possible to share your project in swift 4?
